ansible_events_ui/db/migrations/script.py.mako

```diff
@@ -8,6 +8,7 @@ Create Date: ${create_date}
 from alembic import op
 import sqlalchemy as sa
 ${imports if imports else ""}
+from alembic import context
```

Comment:
I don't think that changes in the template are necessary. Normally database migrations will include only schema migrations, so in my opinion it is redundant to use this template for all generated revisions.